components/Facility/index.js

```diff
@@ -54,62 +47,195 @@ export default function Facility({ facility, setFacilities }) {
+                          id="Fill-3"
+                          fill="#FBBC04"
+                        ></path>
+                        <path
+                          d="M60.0985256,2.08597715 C71.4734541,5.75326622 81.2088401,13.452646 87.0979777,24.8069254 L59.7268409,57.4243229 C59.7268409,57.4243229 63.7768275,52.7163167 63.7768275,46.0266804 C63.7768275,35.9808688 55.317559,28.4769677 46.18031,28.4769677 C37.539329,28.4769677 32.5759616,34.7763351 32.5759616,34.7763351 L32.5759616,14.2276774 L60.0985256,2.08597715 Z"
+                          id="Fill-5"
+                          fill="#4285F4"
+                        ></path>
+                        <path
+                          d="M10.8205677,16.498671 C17.6148264,8.37047717 29.5693077,0 46.0500827,0 C54.0454337,0 60.0701674,2.09864196 60.0701674,2.09864196 L32.5503567,34.7945063 L13.0520525,34.7945063 L10.8205677,16.498671 Z"
+                          id="Fill-7"
+                          fill="#1A73E8"
+                        ></path>
+                        <g
+                          id="Group-11"
+                          transform="translate(0.000000, 16.498740)"
+                        >
+                          <mask id="mask-2" fill="white">
+                            <use xlinkHref="#path-1"></use>
+                          </mask>
+                          <g id="Clip-10"></g>
+                          <path
+                            d="M4.54673637,51.5980964 C4.54673637,51.5980964 -0.000206491502,42.6941829 -0.000206491502,29.7953471 C-0.000206491502,17.6040888 4.73946177,6.94775072 10.8206365,0 L32.5655682,18.2985886 L4.54673637,51.5980964 Z"
+                            id="Fill-9"
+                            fill="#EA4335"
+                            mask="url(#mask-2)"
+                          ></path>
+                        </g>
+                      </g>
+                    </g>
+                  </svg>
+                  <div>
+                    <StyledP>{address.split(",")[0]},</StyledP>
+                    <StyledParagraph>{address.split(",")[1]}</StyledParagraph>
+                  </div>
+                </StyledLink>
+              </ColoredContainer>
+            </ListItem>
+          </div>
         </StyledList>
       </StyledSection>
     </>
   );
 }
 const StyledImage = styled(Image)`
   object-fit: contain;
-  margin-right: 10px;
-  border-radius: 50px;
+  border-radius: 40%;
+`;
+const ImageContainer = styled.div`
+  text-align: center;
 `;
 
 const StyledSection = styled.section`
-  display: flex;
-  border-style: solid;
-  padding: 20px;
-  border-radius: 20px;
-  background-color: white;
+  border-radius: 40px;
+  background-color: var(--secondary-color);
+  box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.1);
+  padding: 30px;
+  margin-bottom: 30px;
```

Comment:
In `StyledImage`  the border-radius is %, in this component the  border-radius is px - please be consistent in using units. 
```suggestion
  border-radius: ???;
  background-color: var(--secondary-color);
  box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.1);
  padding: 30px;
  margin-bottom: 30px;
```